### PR TITLE
StdIO error output simplification

### LINKED ISSUE #119: Error formatting
We can have a method for error handling, which will produce error in a proper format.

### PR DESCRIPTION
**Pull request by maheshgudi**
I have tried to simplify the error output for StdIO type for all languages.
Modified pre-existing test cases.
closes #119 

stdio_output

### CODE REVIEW

File: yaksh/templates/yaksh/view_answerpaper.html [hunk 1]
```diff
@@ -102,7 +103,56 @@
         {% else %}
             <div class="panel panel-danger">
         {% endif %}
-    <div class="panel-heading">Autocheck: {{ answer.error }}</div>
+    <div class="panel-heading">Error:</div>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/view_answerpaper.html]: So this same html is used in number of templates, so can we have a custom tag for this or any other solution to prevent redundancy.

Comment (maheshgudi) [inline yaksh/templates/yaksh/view_answerpaper.html]: I am planning to do this in a separate PR.

**Comment by maheshgudi** [comment]
Resolved all comments.

**Comment by prathamesh920** [comment]
LGTM
